test/files/neg/abstract-report2.check

```diff
@@ -21,15 +21,15 @@ Missing implementations for 13 members. Stub implementations follow:
   def add(x$1: List[_ <: String]): Boolean = ???
   def addAll(x$1: java.util.Collection[_ <: List[_ <: String]]): Boolean = ???
   def clear(): Unit = ???
-  def contains(x$1: Any): Boolean = ???
+  def contains(x$1: Object): Boolean = ???
   def containsAll(x$1: java.util.Collection[_]): Boolean = ???
   def isEmpty(): Boolean = ???
   def iterator(): java.util.Iterator[List[_ <: String]] = ???
-  def remove(x$1: Any): Boolean = ???
+  def remove(x$1: Object): Boolean = ???
   def removeAll(x$1: java.util.Collection[_]): Boolean = ???
   def retainAll(x$1: java.util.Collection[_]): Boolean = ???
   def size(): Int = ???
-  def toArray[T](x$1: Array[T with Object]): Array[T with Object] = ???
+  def toArray[T <: Object](x$1: Array[T]): Array[T] = ???
```

Comment:
Interesting change here, needs to look at this more to decide if its a re- or pro-gression...